### DISCUSSION TITLE
Install Mica For Everyone Error

### DISCUSSION THREAD

**Discussion by knowledgetechid** [MicaForEveryone/MicaForEveryone#73] [Q&A] [answered]
This app package’s publisher certificate could not be verified. Contact your system administrator or the app developer to obtain a new app package with verified certificates. The root certificate and all immediate certificates of the signature in the app package must be verified (0x800B010A)
![Screenshot 2022-06-03 075243](https://user-images.githubusercontent.com/101148765/171764364-c93428de-59c3-45a4-8436-c4868480c504.png)

**Comment by xmine64** [comment, marked as answer]
You need to pay to get a trusted certificate, so I used a self-signed certificate for MFE. Installation guide is here: https://github.com/MicaForEveryone/MicaForEveryone/wiki/Installation
